static/docs/commands-reference/version.md

```diff
@@ -47,6 +48,41 @@ The detail of DVC version depends upon the way of installing the project.
   part is the git commit hash which is one of the commits in the `master` branch
   (also, optional suffix `.mod` means that code is modified).
 
+#### Output of Binary
+
+The detail of Binary depends upon the way of downloading a package.
+
+- **`Binary: True`** - This output is displayed when DVC package is downloaded as
```

Comment:
is downloaded -> is installed?

Reply:
@shcheklein I think is should be downloaded only. But it won't make a difference since `True` gets attached to it when source code is built by `PyInstaller`.

Reply:
it's just the way it's usually described is not about downloading process, but more about the installation process itself. Usually, we say "it installed via PyPi", it's installed from a deb package, etc